src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  selectedTodo: Todo | null;
+  onSelect: (value: Todo | null) => void;
+}
+
+export const TodoItem: FC<Props> = ({
+  todo,
+  selectedTodo,
+  onSelect,
+}) => (
+  <tr
```

Comment:
```suggestion
export const TodoItem: FC<Props> = ({
  todo,
  selectedTodo,
  onSelect,
}) => {
  const isTodoSelected = selectedTodo && selectedTodo.id === todo.id;

  return (
    <tr
```
and use this variable everywhere you need